instant-distance/examples/colors.rs

```diff
@@ -0,0 +1,27 @@
+use instant_distance::{Builder, Search};
+
+fn main() {
+    //
+    let points = vec![Point(255, 0, 0), Point(255, 0, 0), Point(255, 0, 0)];
+    let values = vec!["red", "green", "blue"];
+
+    let map = Builder::default().build(points, values);
+    let mut search = Search::default();
+
+    let cambridge_blue = Point(163, 193, 173);
+
+    let closest_point = map.search(&cambridge_blue, &mut search).next().unwrap();
+
+    println!("{:?}", closest_point.value);
```

Comment:
@djc is this the intended way to retrieve the value? For me, the value changes every time. However, the pointID remains constant. To me it looked like HnswMap.values is sorted prior to being exposed here, but maybe I'm missing something: https://github.com/nrempel/instant-distance/blob/4a4ee8890717db205a38683a11b0c89559d5d480/instant-distance/src/lib.rs#L163-L163

Reply:
Yeah, that should be the way it works. Are you saying this does not print "blue" every time?

Reply:
Correct:
```
nbrempel@Nick-IDS ~/P/instant-distance (main)> cargo run --example colors                    (base) 
   Compiling instant-distance v0.5.0 (/Users/nbrempel/Projects/instant-distance/instant-distance)
    Finished dev [unoptimized + debuginfo] target(s) in 5.92s
     Running `target/debug/examples/colors`
"red"
nbrempel@Nick-IDS ~/P/instant-distance (nrempel/update-readme)> cargo run --example colors   (base) 
    Finished dev [unoptimized + debuginfo] target(s) in 0.04s
     Running `target/debug/examples/colors`
"red"
nbrempel@Nick-IDS ~/P/instant-distance (nrempel/update-readme)> cargo run --example colors   (base) 
    Finished dev [unoptimized + debuginfo] target(s) in 0.04s
     Running `target/debug/examples/colors`
"red"
nbrempel@Nick-IDS ~/P/instant-distance (nrempel/update-readme)> cargo run --example colors   (base) 
    Finished dev [unoptimized + debuginfo] target(s) in 0.04s
     Running `target/debug/examples/colors`
"red"
nbrempel@Nick-IDS ~/P/instant-distance (nrempel/update-readme)> cargo run --example colors   (base) 
    Finished dev [unoptimized + debuginfo] target(s) in 0.04s
     Running `target/debug/examples/colors`
"blue"
nbrempel@Nick-IDS ~/P/instant-distance (nrempel/update-readme)> cargo run --example colors   (base) 
    Finished dev [unoptimized + debuginfo] target(s) in 0.03s
     Running `target/debug/examples/colors`
"red"
nbrempel@Nick-IDS ~/P/instant-distance (nrempel/update-readme)> cargo run --example colors   (base) 
    Finished dev [unoptimized + debuginfo] target(s) in 0.03s
     Running `target/debug/examples/colors`
"red"
nbrempel@Nick-IDS ~/P/instant-distance (nrempel/update-readme)> cargo run --example colors   (base) 
    Finished dev [unoptimized + debuginfo] target(s) in 0.03s
     Running `target/debug/examples/colors`
"red"
nbrempel@Nick-IDS ~/P/instant-distance (nrempel/update-readme)> cargo run --example colors   (base) 
    Finished dev [unoptimized + debuginfo] target(s) in 0.04s
     Running `target/debug/examples/colors`
"red"
nbrempel@Nick-IDS ~/P/instant-distance (nrempel/update-readme)> cargo run --example colors   (base) 
    Finished dev [unoptimized + debuginfo] target(s) in 0.04s
     Running `target/debug/examples/colors`
"green"

```